locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -194,7 +194,7 @@ $ docker images
 
 # Example
 REPOSITORY                      TAG        ID              CREATED
-node                            12         1934b0b038d1    5 days ago
+node                            14         1934b0b038d1    5 days ago
```

Comment:
I'm guessing the ID might be different, but I don't think it matters for the purposes of this example

Reply:
Ya that's what I thought